packages/ckeditor5-html-support/README.md

```diff
@@ -0,0 +1,20 @@
+CKEditor 5 General HTML Support feature
```

Comment:
```suggestion
CKEditor 5 HTML Support feature
```
I'd keep this value synchronized with the name specified in the `LICENSE.md` file.